omnibus/omnibus-test.sh

```diff
@@ -49,7 +49,7 @@ sudo rm -rf "$TMPDIR"
 mkdir -p "$TMPDIR"
 
 # Verify that we kill any orphaned test processes. Kill any orphaned rspec processes.
-ps ax | grep -E 'rspec' | grep -v grep | awk '{ print $1 }' | xargs sudo kill -s KILL || true
+sudo kill -9 $(ps ax | grep -E 'rspec' | grep -v grep | awk '{ print $1 }') || true
```

Comment:
I think `$()` is technically a bash-ism and might fail on older Unixen?  Do we care any more?

Sol9 this certainly wouldn't work on...

Reply:
`ps ax` also fails on solaris i think since BSD-vs-SysV flag syntax, although maybe by Sol11 they've finally supported both?

Reply:
good point will check this on Solaris now

Reply:
The `$()` works on solaris, but `grep -E ` has been failing until now. Removing `-E` here